html/themes/custom/common_design_subtheme/templates/content/node--topic--full.html.twig

```diff
@@ -0,0 +1,193 @@
+{#
```

Comment:
The topic pages are similar to the country, disaster and source pages so the template here should be similar to https://github.com/UN-OCHA/rwint9-site/blob/develop/html/themes/custom/common_design_subtheme/templates/content/taxonomy-term--country--full.html.twig

Reply:
Simplified